src/Components/Auction/__stories__/AuctionRegistrationModal.story.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { AuctionRegistrationModal } from "../AuctionRegistrationModal"
+
+const submitHandler = p => {
+  setTimeout(() => {
+    p.setSubmitting(false)
+    alert("Your Submission Callback Here")
+  }, 1000)
+}
+storiesOf("Components/Auction/AuctionRegistrationModal", module).add(
+  "Default",
```

Comment:
Unless we have other states to show here, I think we can add this story directly under `Components/Auction`:

```typescript
storiesOf("Components/Auction", module).add("AuctionRegistrationModal", () => {
```